### DISCUSSION TITLE
how to take backup in multi-container POD

### DISCUSSION THREAD

**Discussion by vfolk30** [k8up-io/k8up#812] [Q&A]
Hi Experts,
I have a pod which has two  container in a POD.

container 1:  exporter 
container 2:  mysql

I have annotated backup command to perform the application aware backup  but its failing because i dont have mysqldump command in container 1. 
There is way to execute commands in a single container?

**Comment by Kidswiss** [comment]
Hi

It's not currently supported to choose which container it should execute the command in. See #566.

Currently, it will always pick the first container in the array. So if you have full control over the yaml manifest you can put the mysql container at the first position as a workaround.

**Comment by vfolk30** [comment]
Thank @Kidswiss 
If I use this approach , k8up will run the command on second container and it will  fail. 
As per my understating , k8up output will be in failed status for job 

example:
unable to start k8up: backup of annotated pods failed: backup commands failed: cmd.Wait() err: 1

**Comment by Kidswiss** [comment]
I see, then you'll need to have a bit more patience. Implementing #566 is in the current sprint, so it should be available soonish.

**Reply by vfolk30** [reply]
Thank you @vfolk30  #566  will be add good value. 
Can you share the link of scrum board to see  dev task .

**Reply by Kidswiss** [reply]
Currently, our scrum board we're actively using is internal only as there are other projects we work on. We haven't updated the projects board here in Github as it's quite a bit of effort to keep them in sync.
